aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
Prob also should change the heading, as it's more "middleware" than "module," and I'm not quite feel'in "injected," although I do see your point. Prob should say something like any of these ...

> IIS Integration middleware not injected
> IIS Integration middleware not included
> IIS Integration middleware not present
> IIS Integration middleware not provided
> IIS Integration middleware not furnished
> IIS Integration middleware not implemented
> IIS Integration middleware not supplied

Given what we see so much in docs generally, I think "implemented" is common and easily understood for devs.